src/Database/Attach/File.php

```diff
@@ -105,7 +108,10 @@ public function fromFile($filePath)
         $this->file_size = $file->getSize();
         $this->content_type = $file->getMimeType();
         $this->disk_name = $this->getDiskName();
-
+        $realpath = ( empty(trim($file->getRealPath())) 
```

Comment:
Same thing as above. it's just checking if hte real path is empty, if not, recompose it.